src/utils/config.js

```diff
@@ -2,7 +2,7 @@ const environment = process.env.REACT_APP_ENV ?? 'prod';
 
 export function getConfig() {
   const commonConfig = {
-    gooddollar_api: 'https://back.justfortestinc.site',
+    gooddollar_api: 'https://staging.justfortestinc.site',
```

Comment:
this will be different for mainnet right?
staging means for testnet
then we should remove it from `commonConfig`, and assign it separate values below

Reply:
> 

are you sure it's for different environments?
Until yesterday, we used 1 reference